### PR TITLE
Update analytics.md

### PR DESCRIPTION
**Pull request by niva-cumulocity**
Rewrite from PM lead

### CODE REVIEW

File: content/concepts/concepts-introduction-bundle/analytics.md [hunk 1]
```diff
@@ -5,8 +5,9 @@ title: Analytics
...
+For this, {{< product-c8y-iot >}} comes with real-time data streaming and processing capabilities, allowing businesses to process the incoming data and turn it into insights or actions. The predefined [Smart Rules](/cockpit/smart-rules/) offers a wizard-driven approach to get started with data analytics, whereas the [Streaming Analytics application](/streaming-analytics/introduction-analytics/#home) offers flexibility to create even more rules, ranging from simple data transformations to more advanced business metric calculations like uptime or utilisation rate, either for individual devices & assets or aggregated over a sub-set of the entire connected fleet. Real-time rules also allow for both triggering external workflows (for example, creation of a service ticket) as well as sending operations back to the equipment to, for example, optimize machine operations based on analytical insights.
```
Comment (BeateRixen) [inline content/concepts/concepts-introduction-bundle/analytics.md]:
```suggestion
For this, {{< product-c8y-iot >}} comes with real-time data streaming and processing capabilities, allowing businesses to process the incoming data and turn it into insights or actions. The predefined [smart rules](/cockpit/smart-rules/) offer a wizard-driven approach to get started with data analytics, whereas the [Streaming Analytics application](/streaming-analytics/introduction-analytics/#home) offers flexibility to create even more rules, ranging from simple data transformations to more advanced business metric calculations like uptime or utilization rate, either for individual devices & assets or aggregated over a subset of the entire connected fleet. Real-time rules also allow for both triggering external workflows (such as the creation of a service ticket) as well as sending operations back to the equipment, for example, to optimize machine operations based on analytical insights.
```

File: content/concepts/concepts-introduction-bundle/analytics.md [hunk 2]
```diff
@@ -5,8 +5,9 @@ title: Analytics
...
 
-[{{< product-c8y-iot >}} DataHub](/datahub/datahub-overview/) makes it easy for data analysts and data scientists to derive insights from the equipment data by allowing for high-performant analytical queries. The support for SQL via standard ODBC/JDBC interfaces allow for a seamless integration with any data analytics tool and workflow.
+Not all metrics and insights are solely relying on real-time incoming data: metrics like Remaining Useful Life of a device or asset include some kind of benchmark or learning based on historical data from that same or from similar devices or assets, and that is where AI and Machine Learning come in. The [{{< product-c8y-iot >}} DataHub](/datahub/datahub-overview/) makes it easy for data analysts and data scientists to access the equipment data which is required to create and/or train any kind of AI model; the support for SQL via standard ODBC/JDBC interfaces allow for a seamless integration with the data science workbench or open source tooling of choice.
```
Comment (BeateRixen) [inline content/concepts/concepts-introduction-bundle/analytics.md]:
```suggestion
Not all metrics and insights are solely relying on real-time incoming data. Metrics like "remaining useful life" of a device or asset include some kind of benchmark or learning based on historical data from that same or from similar devices or assets, and that is where AI and Machine Learning (ML) come in. The [{{< product-c8y-iot >}} DataHub](/datahub/datahub-overview/) makes it easy for data analysts and data scientists to access the equipment data which is required to create and/or train any kind of AI model; the support for SQL via standard ODBC/JDBC interfaces allow for a seamless integration with the data science workbench or open source tooling of choice. 
```

File: content/concepts/concepts-introduction-bundle/analytics.md [hunk 3]
```diff
@@ -5,8 +5,9 @@ title: Analytics
...
+
+Bear in mind that the real value of AI/ML comes with the operationalization of the created models: you need to close the loop onto the incoming IoT data and switch from historical insight generation to real-time insight generation. Here again does our [Streaming Analytics application](/streaming-analytics/introduction-analytics/#home) come into play to support you in orchestrating the flow of data towards the deployed model and afterwards working with the model output to, for example, raise an alarm of trigger a workflow. For more information on machine learning capabilities, refer to the [Machine Learning introduction](/machine-learning/ml-introduction/) in the user documentation.
```
Comment (BeateRixen) [inline content/concepts/concepts-introduction-bundle/analytics.md]:
```suggestion
Bear in mind that the real value of AI/ML comes with the operationalization of the created models. You need to close the loop onto the incoming IoT data and switch from historical insight generation to real-time insight generation. Here again does our [Streaming Analytics application](/streaming-analytics/introduction-analytics/#home) come into play to support you in orchestrating the flow of data towards the deployed model and afterwards working with the model output to, for example, raise an alarm of trigger a workflow. For more information on machine learning capabilities, refer to the [Machine Learning introduction](/machine-learning/ml-introduction/).
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@neuerburg Please review and approve

**Review by neuerburg — APPROVED** [review]

**Comment by BeateRixen** [comment]
@niva-cumulocity Can I merge this?

**Comment by niva-cumulocity** [comment]
Yes, can be merged!



Kind regards,



Nick Van Damme

Director Product Management - IoT Analytics


As of December 1st, my email address will change!

Current: ***@***.*** (ceasing to exist)

New: ***@***.*** (please use for all future communication)



[cid:92acd884-b1cc-4b5c-bc75-e51e1f391f40]<https://www.softwareag.com/en_corporate/resources/iot/wp/power-of-connected-equipment.html?utm_source=empower&utm_medium=email&utm_region=hq&utm_campaign=iot_smart-products&utm_subcampaign=empower_email_sig&utm_content=wp_power_of_connected_equipment>


________________________________

**Pull request merged**